webextensions/api/privacy.json

```diff
@@ -75,6 +75,30 @@
                 }
               }
             }
+          },
+          "HTTPS-only": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "84"
+                },
+                "firefox_android": {
+                  "version_added": false
```

Comment:
This API is also available on Firefox for Android (but there is no visible UI to enable it, so it's currently always "never").

(On all platforms the setting is read-only)
(On Fenix Nightly users can visit about:config to toggle the underlying preferences, but that's not really an expected scenario)